UI_oM/ParamInfo.cs

```diff
@@ -51,6 +51,8 @@ public class ParamInfo : BHoMObject
 
         public virtual string DefaultValueWarning { get; set; } = "";
 
+        public virtual bool IsExposed { get; set; } = true;
```

Comment:
I think there is overlap between this new property and the `IsSelected` property, which essentionally does the same thing, at least on the level of Grasshopper (IsSelected == false -> input not on the component).

Might be better to try to align the two in this case, also to avoid having to re-implement the running mechanisms, fetching default values, etc.

Basically, a bit scared of the overlap, and a simple solution might just be to remove this property and simply set the IsSelected instead, which might just finish the job without the need for modification in any other part of the code, like for example GH.

Reply:
Have done - per our offline discussion, we're happy for advanced/super users to allow `hidden` inputs to be visible in the philosophy of help (by hiding) without constraining (making inaccessible).